src/content/configuration/experiments.md

```diff
@@ -17,6 +17,12 @@ W> Because experimental features have relaxed semantic versioning and might cont
 Available options:
 
 - `mjs`: Support `.mjs` files as a way to define [EcmaScript modules](https://nodejs.org/api/esm.html#esm_ecmascript_modules)
+- `syncWebAssembly`: Support old WebAssembly like in webpack 4
+- `asyncWebAssembly`: Support new WebAssembly according to the [updated spec](https://github.com/WebAssembly/esm-integration), makes a WebAssembly module an async module
+- `topLevelAwait`: Support [Top Level Await Stage 3 proposal](https://github.com/tc39/proposal-top-level-await), makes the module an async module when `await` is used on top-level
+- `importAsync`: import async modules with `import`
+- `importAwait`: import async modules with `import await`
+- `asset`: a type of module that allows to use asset files (fonts, icons, etc) without configuring additional loaders, similar to `file-loader`|`url-loader`|`raw-loader`
```

Comment:
I think icons is the use case of images, better to list actual resource type. Also added spaces near pipes to make it less cryptic for the crawlers
```suggestion
- `asset`: a type of module that allows to use asset files (fonts, images, etc) without configuring loaders to handle their importing, similar to `file-loader` | `url-loader` | `raw-loader`
```